doc/source/faq.rst

```diff
@@ -0,0 +1,61 @@
+
+
+Frequently asked questions
+===========================
+
+What is the difference between chatmail relays and classic e-mail servers?
+--------------------------------------------------------------------------
+
+A chatmail relay is a minimal Mail Transport Agent (MTA) setup that
+goes beyond what classic e-mail servers offer:
+
+-  **Zero State:** no private data or metadata collected, messages are auto-deleted, low disk usage
+
+-  **Instant/Realtime:** sub-second message delivery, realtime P2P
+   streaming, privacy-preserving Push Notifications for Apple, Google, and Huawei;
```

Comment:
```suggestion
   streaming, privacy-preserving Push Notifications for Apple, Google, and Ubuntu Touch;
```

No Huawei currently, but we have Ubuntu Touch.
Can have a link to https://docs.ubports.com/en/latest/appdev/guides/pushnotifications.html

Reply:
fixed